qiskit/assembler/assemble_schedules.py

```diff
@@ -87,18 +103,13 @@ def assemble_schedules(schedules, qobj_id, qobj_header, run_config):
                 # add samples to pulse library
                 user_pulselib[name] = instruction.command
 
-            elif isinstance(instruction, AcquireInstruction):
-                max_memory_slot = max(max_memory_slot,
-                                      *[slot.index for slot in instruction.mem_slots])
-                if meas_map:
-                    # verify all acquires satisfy meas_map
-                    _validate_meas_map(instruction, meas_map)
-
             converted_instruction = instruction_converter(shift, instruction)
             qobj_instructions.append(converted_instruction)
 
         # experiment header
+        # TODO: check if these items are enough for other qiskit modules, e.g. ignis RB
```

Comment:
because experimental header of `QuantumCircuit` contains those things.
https://github.com/Qiskit/qiskit-terra/blob/7bc44b83aab1a0e25ed2a3c9301465a06f8210ad/qiskit/assembler/assemble_circuits.py#L63-L69

On the other hand, pulse experimental header has only `name` (and `memory_slots` will be added with this PR).

My point is: the result qobj format of pulse experiment with `meas_level=2` is basically identical to those of circuit experiments. The result can be supplied to other qiskit modules (post processing) targeting circuit experiment - randomized benchmark of Ignis is an good example. If those modules require other header items, such post processing with pulse result will be failed. but I cannot check everything outside terra.

Reply:
I think checking all modules doesn't make sense because we should continue to monitor all modules will be added in future. How about making default value policy for pulse module?

`n_qubits`: from backend configuration
`qubit_labels` : `q0`, `q1`, `q2` ....
`qreg_sizes`: same as `n_qubits`, but no grouping (only `q*` series)
`clbit_labels`: `c0`, `c1`, `c2` ....
`creg_sizes`: same as `memory_slots`, but no grouping (only `c*` series)

using above policy, we can generate `QuantumCircuit` compatible experimental header.

Reply:
right. I'll raise another issue to discuss that.